release-notes/2024-09/README.md

```diff
@@ -0,0 +1,221 @@
+    - [View this service configuration](https://github.com/ni/install-systemlink-enterprise/blob/2024-09/getting-started/templates/systemlink-values.yaml#L469)
+- Adjust how many alarms you can create in the SystemLink Enterprise Alarm Service - [Configuring Alarm Limits](https://www.ni.com/docs/en-US/bundle/systemlink-enterprise/page/configuring-alarm-limits.html).
+    - [View this service configuration](https://github.com/ni/install-systemlink-enterprise/blob/2024-09/getting-started/templates/systemlink-values.yaml#L484)
+- Learn about the [performance metrics for the Dashboard Host Alarm Service](https://www.ni.com/docs/en-US/bundle/systemlink-enterprise/page/performance-metrics-dashboard-host-service.html).
+- Privilege **List published notebooks in SystemLink and download published notebooks into the Jupyter development environment** was split to **List available notebooks** to list the published notebooks, and **List published notebooks in SystemLink and download published notebooks into the Jupyter development environment** to list and download notebooks for notebook development.
+    - This privilege is enabled by default for Collaborator, Data Maintainer, and Systems Maintainer built-in roles.
+    - For the Automated Agent, the **List published notebooks in SystemLink and download published notebooks into the Jupyter development environment** privilege will be removed, as the Automated Agent should not need to download the notebooks content.
+- Support for [Private Certificate Authorities](https://www.ni.com/docs/en-US/bundle/systemlink-enterprise/page/private-certificate-authorities.html).
+    - Connect to MongoDB and S3 compatible interfaces that use certificates signed by a private authority.
+    - Call SystemLink APIs from Jupyter Notebook when the SystemLink API endpoint is configured to use a certificate signed by a private authority.
+    - Trusted certificates specified at `global.trustedCertificatesSecrets` are now installed on servers that use Jupyter. These certificates are used when calling SystemLink or other external APIs from notebooks.
+    - Trusted certificates specified at `global.trustedCertificatesSecrets` are now installed on execution pods. These certificates are used when calling SystemLink or other external APIs from the pods.
+- Added abilities to configure the rate-limit values for notebook APIs inside the Helm chart.
+- Notebook Execution Service
+    - Created a dedicated swagger page for Artifact APIs.
+    - Implemented Rate limiting for the Artifact APIs.
+    - Created S3 buckets for Artifact APIs.
+- Notebook Execution Service, Systems Management Service
+    - Added support for deploying a custom CA root to all managed clients.
+        - [View this service configuration](https://github.com/ni/install-systemlink-enterprise/blob/2024-09/getting-started/templates/systemlink-values.yaml#L119)
```

Comment:
We should add a line here the mimic the structure and content included when discussion private CA support for `global.trustedCertificateSecrets` (line 22)

Reply:
I did rephrase this as below.
@stefanc18 Stefan Ciuprina, could you help review the 1st and 2nd line below - specifically to the 2nd line could you help suggest the right words. Thanks,

- Systems Management Service support for deploying a custom CA root to all managed clients.
    - Trusted certificates specified at `global.trustedCertificatesSecrets` are now installed on containers that run Jupyter notebooks. These certificates are used when calling SystemLink or other external APIs from notebooks.
    - [View this service configuration](https://github.com/ni/install-systemlink-enterprise/blob/2024-09/getting-started/templates/systemlink-values.yaml#L119)

Reply:
The wording looks good to me, but I see that it is mentioning custom certificates on Jupyter notebooks as a subsection of "Systems Management Service support for deploying a custom CA root to all managed clients", which are unrelated. This exact phrase seems to be already written on line 22.

Moreover, in the context of Systems Management Service, custom certificates deployed to managed clients is not done through "global.trustedCertificatesSecrets", but by setting `global.deployApiHostCertificateToSystems` to `true` and providing a single certificate secret at `global.apiHostCertificateSecret`.

Reply:
I reworked all of these for clarity and accuracy